lib/subordinateio.c

```diff
@@ -1117,6 +1117,15 @@ bool release_subid_range(struct subordinate_range *range, enum subid_type id_typ
 	return ret;
 }
 
+void free_subid_pointer(void *ptr) {
```

Comment:
Opening brace on its own line.  See <https://www.kernel.org/doc/html/latest/process/coding-style.html#placing-braces-and-spaces>.

Reply:
Thanks.

Reply:
BTW What do you think about `clang-format` check in linting stage in GitHub CI? There is [a config file][1] which follows kernel code style guidelines. Then just run something like the following.

```shell
clang-format -i --dry-run -Werror lib/subordinateio.c 2> /dev/null
```

[1]: https://docs.kernel.org/process/clang-format.html

Reply:
That would cry too much with the existing code base.  In the far far future, when I've slowly made the style consistent, yeah, that would help.  :)